doc/wiki/Developing-with-VS-Code.md

```diff
@@ -0,0 +1,52 @@
+# Developing Gamer with Visual Studio Code
+
+This guide provides step-by-step instructions on how to set up and use Visual Studio Code (VS Code) for developing the Gamer codebase.
+
+## Setup
+
+### Prerequisites
+
+- **Quick Start**: Follow the instructions in the [[Quick Start | Quick-Start]] guide to download Gamer and do at least one [[demo | Quick-Start:-1D-Shock-Tube]].
+- **Visual Studio Code**: Download and install from [https://code.visualstudio.com/](https://code.visualstudio.com/).
+- **C/C++ Extension**: Install the "C/C++" extension from the VS Code Marketplace.
+
+### Setting up the workspace
+
+1. **Launch VS Code**.
+2. **Open the Gamer Project Folder**:
+   - Go to `File` > `Open Folder...`.
+   - Select your Gamer project directory.
+
+### Disabling auto-formatting
+
+To disable auto-formatting in VS Code, add the following to your `settings.json`:
+
+Access `settings.json` by navigating to `File` > `Preferences` > `Settings`, then search for `C_Cpp.formatting` and set it to `"disabled"`.
+
+### Configuring VS Code to integrate with Gamer
+
+Run the following script from the root directory of the Gamer project:
+```bash
+sh tool/vscode/copy_to_vscode.sh
+```
+This script copies the necessary configuration files to the `.vscode` directory, integrating Gamer with VS Code. This script will ask you to enter the working directory under `bin/` where the executables are located. These paths of working directories are located in the `launch.json` and `tasks.json` files.
+
+## Developing with VS Code
+
+### Run build task
+
+After configuring Gamer with [configure.py](https://github.com/gamer-project/gamer/wiki/Installation%3A-Configure.py), select `Terminal` > `Run Build Task...` or press `Ctrl + Shift + B` to start the build task. This will update the macros, providing IntelliSense highlighting support.
```

Comment:
It seems that debugging with parallel at `vscode` is difficult (not only `mpi`, but also `openmp`). Should we remind to disable `mpi` and `openmp` here?